example_flutter/lib/main.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:const_date_time/const_date_time.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  const title = 'Flutter Web Demo';
+  final now = DateTime.now();
+  runApp(MaterialApp(
+    title: title,
+    home: Scaffold(
+      appBar: AppBar(
+        title: const Text(title),
+      ),
+      body: Center(
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Text(now.difference(DateTime(0)).toString()),
```

Comment:
Just an FYI, a workaround might be swapping the parameters and negating the value:
```dart
Text(-(const ConstDateTime(0)).difference(now).toString()),
```

Reply:
In my code, I mix ConstDateTime and DateTime, and there is no way to guarantee the order of the calls.